scss/bootstrap/overrides/style/_forms.scss

```diff
@@ -75,6 +89,36 @@ label {
   }
 }
 
+// Style new group of type radio
+.form-inline {
+
+  &.form-inline-tabs .form-group {
+    box-shadow: $crm-form-control-shadow;
+    border: 1px solid $crm-darker-grayblue;
+    line-height: 1em;
+    padding: 8px 15px;
+    margin-right: -5px;
```

Comment:
This is to remove the spacing between elements. Without marging right it would be like this 

![screen shot 2017-04-06 at 11 05 15 am](https://cloud.githubusercontent.com/assets/2423218/24746344/16328886-1ab9-11e7-92c4-36574b53c908.png)

Between the target to be like this 

![screen shot 2017-04-06 at 11 05 37 am](https://cloud.githubusercontent.com/assets/2423218/24746380/2e7de34a-1ab9-11e7-8e08-ae3d3a947da7.png)


So giving the groups a float:left; OR margin-right:-5px should resolve this